lib/src/db/db.dart

```diff
@@ -265,12 +277,11 @@ class Query<T extends Model> {
 class DatastoreDB {
   final ds.Datastore datastore;
   final ModelDB _modelDB;
-  Partition _defaultPartition;
+  final Partition _defaultPartition;
 
-  DatastoreDB(this.datastore, {ModelDB modelDB, Partition defaultPartition})
-      : _modelDB = modelDB ?? ModelDBImpl() {
-    _defaultPartition = defaultPartition ?? Partition(null);
-  }
+  DatastoreDB(this.datastore, {ModelDB? modelDB, Partition? defaultPartition})
+      : _modelDB = modelDB ?? ModelDBImpl(),
+        _defaultPartition = defaultPartition ?? Partition(null);
```

Comment:
Nit: `Partition.DEFAULT`?

Reply:
I think that's `datastore.Partition` and not `db.Partition`, yes, I'm also confused..